raiden/ui/app.py

```diff
@@ -143,6 +143,7 @@ def run_app(
     resolver_endpoint: str,
     routing_mode: RoutingMode,
     config: Dict[str, Any],
+    **kwargs: Dict,
```

Comment:
why wouldn't `kwargs` be `Dict[str, Any]` ?

Reply:
and how is the `kwargs` used?

Reply:
It's not, btu removing it breaks the smoketest: https://github.com/raiden-network/raiden/issues/4053